tests/gen_cert_test.py

```diff
@@ -105,3 +111,16 @@ def test_cert_upload():
     r = urllib2.urlopen(download_url)
     with tempfile.NamedTemporaryFile(delete=True) as f:
         f.write(r.read())
+
+
+def test_render_flair_function():
+    """Make sure flair rendering function behaves properly"""
+    cert = CertificateGen(settings.CERT_DATA.keys()[0])
+    overlay_pdf_buffer = StringIO.StringIO()
+    page = canvas.Canvas(overlay_pdf_buffer, pagesize=landscape(A4))
+
+    flair = []
+    assert_false(draw_flair(cert, flair, 'top', page))
+
+    flair = [{'image': {'x': 575, 'y': 325, 'width': 125, 'height': 125, 'file': 'images/test_flair.png'}}]
```

Comment:
We should split this into 2 separate test methods.

Each `test_` method should test just one thing.

```python
def test_draw_flair_expect_true():
    assert_true(...)

def test_draw_flair_expect_false():
    assert_false(...)
```

My method names could be better, but you get the idea..